website/content/docs/connect/config-entries/ingress-gateway.mdx

```diff
@@ -137,6 +287,7 @@ kind: IngressGateway
 metadata:
   name: us-east-ingress
   namespace: default
+  partition: team-frontend
```

Comment:
From the last discussion, partition isn't part of `metadata`, it can only be in the `spec`
```suggestion
```